apisix/admin/v3_adapter.lua

```diff
@@ -135,9 +136,24 @@ local function filter(body, args)
         end
 
         if args.label then
-            local matched = re_find(body.list[i].value.label, args.label, "jo")
-            if not matched then
-                label_matched = false
+            if body.list[i].value.label then
```

Comment:
Is there somewhere we use label to mark a resource? AFAIK, we only have `labels`, no `label`

Reply:
updated